public/sw.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+const VERSION = 1
+const CACHE = 'crime-data-explorer'
+
+self.addEventListener('install', event => {
```

Comment:
any advantage to using `self` instead of `window`? have you seen this? https://stackoverflow.com/a/3217907